modules/web-console-configuration.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="web-console-configuration_{context}"]
+= Configuring the web console
+
+You can configure image registry settings by editing the
+`image.config.openshift.io/cluster` resource.
```

Comment:
So there are 2 different configs that can affect the console, you have them combined into one config here:

```bash
oc edit console.config.openshift.io cluster
```
for the top level config (this is the user facing config):

```yaml
apiVersion: config.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec: 
  authentication:
    # you can configure this for single sign-out
    # this comment explains the feature:
    #  - https://github.com/openshift/api/blob/master/config/v1/types_console.go#L44
    logoutRedirect:  https://www.somewhere.com/idp-logout-redirect
status:
  # we report this to you for your convenience. You can't change it,
  # it will just be put back to the correct value.
  consoleURL: https://console-openshift-console.apps.<your-cluster>.com
```

Reply:
@benjaminapetersen, thank you very much for that clarification! I added a separate topic to disable to console. Will you PTAL?

Reply:
Yup! looks good.